numba/core/byteflow.py

```diff
@@ -460,6 +463,13 @@ def op_LOAD_CONST(self, state, inst):
 
     def op_LOAD_ATTR(self, state, inst):
         item = state.pop()
+        if PYVERSION in ((3, 12), ):
+            if inst.arg & 1:
+                state.push(state.make_null())
```

Comment:
Does the case where `self` is involved need to be handled, could Numba extension code reach this?